src/commands/init.js

```diff
@@ -161,7 +161,11 @@ const loadAbiFromEtherscan = async (network, address) =>
         }.etherscan.io/api?module=contract&action=getabi&address=${address}`,
       )
       let json = await result.json()
-      return JSON.parse(json.result)
+      if (json.status === '1' && json.message === 'OK') {
```

Comment:
What's the meaning of `status` and `message` here?

Reply:
I couldn't find documentation on it anywhere but I tried valid and invalid requests and the valid ones returned `status: '1', message: 'OK'`, whereas the invalid ones returned `status: '0', message: 'NOK'`.

The only reference I could find was this:
https://github.com/sebs/etherscan-api/blob/242d85a9e5ef066fcabd315dd5c9f95d656845b4/lib/get-request.js#L39-L40

Reply:
Got it, could you put that in a comment? Possibly we should check just the status?

Reply:
Done!